api/room_resource/schema.py

```diff
@@ -20,6 +23,58 @@ class Meta:
         model = ResourceModel
 
 
+class RoomResource(SQLAlchemyObjectType):
+    class Meta:
+        model = RoomResourceModel
+
+
+class AssignResource(graphene.Mutation):
+    """
+        Assigns a resource to a room
+    """
+    class Arguments:
+        resource_id = graphene.Int(required=True)
+        quantity = graphene.Int(required=True)
+        room_id = graphene.Int(required=True)
+    room_resource = graphene.Field(RoomResource)
+
+    @Auth.user_roles('Admin')
+    def mutate(self, info, **kwargs):
+        validate_empty_fields(**kwargs)
+        query = RoomSQLAlchemyObject.get_query(info)
+        active_room = query.filter(RoomModel.state == "active")
+        exact_room = active_room.filter(
+            RoomModel.id == kwargs['room_id']).first()
+        if not exact_room:
+            raise GraphQLError("Room not found")
+        exact_resource = ResourceModel.query.filter_by(
+            id=kwargs['resource_id']).first()
+        if not exact_resource:
+            raise GraphQLError('invalid resource id')
```

Comment:
What is the different between `Invalid resource id` and `Resource with such id does not exist`?